pkg/scheduler/controller/shoot/reconciler.go

```diff
@@ -235,7 +262,7 @@ func filterSeedsForZonalShootControlPlanes(seedList []gardencorev1beta1.Seed, sh
 	return seedList, nil
 }
 
-func applyStrategy(shoot *gardencorev1beta1.Shoot, seedList []gardencorev1beta1.Seed, strategy config.CandidateDeterminationStrategy) ([]gardencorev1beta1.Seed, error) {
+func applyStrategy(log logr.Logger, shoot *gardencorev1beta1.Shoot, seedList []gardencorev1beta1.Seed, strategy config.CandidateDeterminationStrategy, schedulerRegionConfig *corev1.ConfigMap) ([]gardencorev1beta1.Seed, error) {
```

Comment:
We could call it `regionConfigMap` everywhere since we already are in "the scheduler" :)